python/ray/tests/test_typing.py

```diff
@@ -7,14 +7,12 @@
 TYPING_TEST_DIRS = os.path.join(os.path.dirname(__file__), "typing_files")
 
 
-@pytest.mark.skipif(sys.platform == "win32", reason="Failing on Windows.")
 def test_typing_good():
```

Comment:
`test_typing_good` and `test_typing_bad` don't seem to be working on the windows CI at the moment :)

Reply:
Let me check again.

Reply:
Ok yeah, why don't you try to fix it? You can run bazel locally and can see where the file is and set up the path in such a way that it can be found :)